7.1/contrib/etc/conf.d/php-fpm/fpm.conf

```diff
@@ -23,7 +23,7 @@ pid = /tmp/php-fpm.pid
 ; If it's set to "syslog", log is sent to syslogd instead of being written
 ; in a local file.
 ; Default Value: /var/log/php-fpm.log
-error_log = /tmp/fpm-error.log
+error_log = /proc/self/fd/2
```

Comment:
12factor.net best practice of logging to stderr. This is also what the official php-fpm image does as per https://serverfault.com/a/738509/216976